PWGDQ/Tasks/dqFlow.cxx

```diff
@@ -0,0 +1,396 @@
+    //      }
+    //      return;
+    //    }
+  }
+
+  // Templated function instantianed for all of the process functions
+  template <uint32_t TEventFillMap, uint32_t TTrackFillMap, typename TEvent, typename TTracks>
+  void runFillQvector(TEvent const& collision, aod::BCs const& bcs, TTracks const& tracks1)
+  {
+    // Reset the fValues and Qn vector array
+    VarManager::ResetValues(0, VarManager::kNVars);
+    // Fill the event information with the VarManager
+    VarManager::FillEvent<TEventFillMap>(collision);
+
+    uint64_t tag = 0;
+    // store the selection decisions
+    for (int i = 0; i < kNsel; i++) {
+      if (collision.selection()[i] > 0) {
+        tag |= (uint64_t(1) << i);
+      }
+    }
+    if (collision.sel7()) {
+      tag |= (uint64_t(1) << kNsel); //! SEL7 stored at position kNsel in the tag bit map
+    }
+    int mult = tracks1.size();
+
+    // TODO: properly access to config files from ccdb using bc.timestamp()
+    // auto bc = collision.bc_as<aod::BCsWithTimestamps>();
+
+    if (fcfgAcceptance.value.empty() == false) {
+      // cfg.mAcceptance = ccdb->getForTimeStamp<GFWWeights>(fcfgAcceptance.value, bc.timestamp());
+      if (cfg.mAcceptance) {
+        LOGF(info, "Loaded acceptance histogram from %s (%p)", fcfgAcceptance.value.c_str(), (void*)cfg.mAcceptance);
+      } else {
+        LOGF(warning, "Could not load acceptance histogram from %s (%p)", fcfgAcceptance.value.c_str(), (void*)cfg.mAcceptance);
+      }
+    }
+    if (mult < 1) {
+      return;
+    }
+    if (!collision.sel7()) {
+      return;
+    }
+    float vtxz = collision.posZ();
+
+    fGFW->Clear();
+    const auto centrality = collision.centRun2V0M();
+    if (centrality > 100) {
+      return;
+    }
+
+    float l_Random = fRndm->Rndm(); // used only to compute correlators
+    float weff = 1.0, wacc = 1.0;   // acceptance and efficiency weights
+
+    // Fill the GFW object in the track loop
+    for (auto& track : tracks1) {
+
+      if (cfg.mEfficiency) {
+        weff = cfg.mEfficiency->GetBinContent(cfg.mEfficiency->FindBin(track.pt()));
+      } else {
+        weff = 1.0;
+      }
+      if (weff == 0) {
+        continue;
+      }
+      weff = 1. / weff;
+      if (cfg.mAcceptance) {
+        wacc = cfg.mAcceptance->GetNUA(track.phi(), track.eta(), vtxz);
+      } else {
+        wacc = 1;
+      }
+      // Fill the track information in the VarManager
+      VarManager::FillTrack<TTrackFillMap>(track);
+
+      int ptin = 0; // default value
+      int mask = 3; // default value
+      // Fill the GFW for each track in order to compute Q vector
+      fGFW->Fill(track.eta(), ptin, track.phi(), wacc * weff, mask);
+    }
+
+    bool fillFlag = kFALSE;    // could be used later
+    bool DQEventFlag = kFALSE; // could be used later
+    for (unsigned long int l_ind = 0; l_ind < corrconfigs.size(); l_ind++) {
+      FillFC(corrconfigs.at(l_ind), centrality, l_Random, fillFlag, DQEventFlag);
+    };
```

Comment:
since FillFC has all the code commented out, I suggest to comment out this block as well